test/integration/node-specific/examples/transactions.test.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const { MongoClient } = require('../../../mongodb');
+const { MongoClient } = require('mongodb');
```

Comment:
Those files (inside examples/ folder) are used by the docs team. I don't know if changes like this should also be synced with them.
This notation (`const { MongoClient } = require('mongodb');`) is used in all other files, and the only 2 exceptions were this file and `versioned_api.js`.